src/infrastructure/database/repositories/category_repository.py

```
@@ -0,0 +1,132 @@
+        from src.infrastructure.database.models.project import ProjectModel
+        stmt = (
+            select(CategoryModel)
+            .join(ProjectModel)
+            .where(
+                and_(
+                    ProjectModel.owner_id == UUID(user_id),
+                    CategoryModel.name == name
+                )
+            )
+        )
+        
```

Ruff (0.11.9)

Blank line contains whitespace

Remove whitespace from blank line

(W293)